components/collective-page/sections/SponsorsAndParticipants.js

```diff
@@ -68,6 +70,14 @@ const Participants = ({ collective: event, LoggedInUser }) => {
 
   const canEditEvent = LoggedInUser && LoggedInUser.canEditEvent(event);
 
+  React.useEffect(() => {
+    if (canEditEvent) {
+      console.log('refetching');
+      refetch();
+      setIsRefetched(true);
```

Comment:
If we don't `await` the `refetch`, this will be set immediately. We only want to show the Export CSV link once the data is refetched, not before.